.github/workflows/test.yml

```
@@ -23,6 +23,10 @@ jobs:
 
     - name: Setup .NET
       uses: actions/setup-dotnet@v4
+      with:
+        dotnet-version: | 
+          6.x
+          8.x
```

```suggestion
      with:
        dotnet-version: 8.x 
```

Only 8.x is needed here

---

The CI failed with https://github.com/qdrant/qdrant-dotnet/actions/runs/12782670159/job/35632531371

---

hmm, that's weird. There's built-in backward compatibility support so this shouldn't be an issue. Maybe try with just net8.0, and if it's still an issue, add 6.0?

---

The TargetFramework in Build.csproj can be updated to net8.0

---

> Maybe try with just net8.0, and if it's still an issue, add 6.0?

That was the original config though.

---

Resolved in: https://github.com/qdrant/qdrant-dotnet/pull/78#discussion_r1917471402